### PR TITLE
[Bug 1015564] Add Mozillians.org profile link into SUMO profile

### PR DESCRIPTION
**Pull request by safwanrahman**
This Will make a User/Contributor to add mozillians profile link in their profile

### CODE REVIEW

**Comment by rlr** [comment]
I wonder if we should ask for the Mozillians username instead of the URL. What do you think?

**Comment by safwanrahman** [comment]
I think link is Better. So if anyone want to see their Mozillians profile, they justclick on the Link and go!
If we add the Mozillians Username, it will be time consuming to find out anyones profile!

Or we Can do as following
User will add only Mozillians URL, but it will make a link redirect to their Mozillians Profle. What do you think?

**Comment by rlr** [comment]
@safwanrahman if they provide the username, then we can create the URL and link them properly because we know what the URLs look like. I just think it's less confusing that way.

Also, travis doesn't like something.

**Comment by mythmon** [comment]
Travis failed because of a linting error:

```
$ scripts/lint.sh
kitsune/users/models.py:79:34: E128 continuation line under-indented for visual indent
```

**Comment by safwanrahman** [comment]
@rlr @mythmon I think its Ok now. and Travis pass also. 
r?

File: kitsune/users/templates/users/mobile/profile.html [hunk 1]
```diff
@@ -80,6 +80,10 @@
           <header>{{ _('Facebook') }}</header>
           <a class="detail" rel="nofollow" href="{{ profile.facebook }}">{{ profile.facebook }}</a>
         {% endif %}
+        {% if profile.mozillians %}
+          <header>{{ _('Mozillians') }}</header>
+          <a class="detail" rel="nofollow" href="http://mozillians.org/u/{{profile.mozillians}}">{{ profile.mozillians }}</a>
```
Comment (rlr) [inline kitsune/users/templates/users/mobile/profile.html]: You should link straight to https and save the redirect.

Comment (rlr) [inline kitsune/users/templates/users/mobile/profile.html]: Also, it looks like they like the trailing slash: https://mozillians.org/en-US/u/safwan/

Add a `/` to the end and save another redirect.

File: kitsune/users/templates/users/profile.html [hunk 2]
```diff
@@ -125,6 +125,12 @@ <h2 class="location">
                 <a rel="nofollow" href="{{ profile.facebook }}">{{ profile.facebook }}</a>
               </li>
             {% endif %}
+            {% if profile.mozillians %}
+              <li>
+                <label>{{ _('Mozillians') }}</label>
+                <a rel="nofollow" href="http://mozillians.org/u/{{profile.mozillians}}">{{ profile.mozillians }}</a>
```
Comment (rlr) [inline kitsune/users/templates/users/profile.html]: same here

**Comment by rlr** [comment]
Some minor comments. I like it this way!

**Comment by rlr** [comment]
Awesome. Gonna test it now.

**Comment by rlr** [comment]
works great. r+

**Comment by rlr** [comment]
24b03fbe523a55c361c53ff3d5393e9711caa8d3 [Bug 1015564] Add Mozillians.org profile link into SUMO profile

**Comment by rlr** [comment]
Thank you @safwanrahman !

**Pull request closed without merging**